index.html

```diff
@@ -9,7 +9,9 @@
 </head>
 <body>
   <div class="book-container">
-    <img src="./assets/images/book-cover.png" alt="book-cover" class="book-cover">
+    <div class="book-cover-image">
+      <img src="./assets/images/book-cover.png" alt="book-cover" class="book-cover">
+    </div>
```

Comment:
use css for add images and reduce html use.

Reply:
ready solved